api/config/usecase/manager.go

```diff
@@ -0,0 +1,74 @@
+package usecase
+
+import (
+	"github.com/monitoror/monitoror/models"
+	"github.com/monitoror/monitoror/pkg/monitoror/builder"
+	"github.com/monitoror/monitoror/pkg/monitoror/utils"
+)
+
+type (
+	ConfigData struct {
+		tileConfigs        map[models.TileType]map[string]*TileConfig
+		dynamicTileConfigs map[models.TileType]map[string]*DynamicTileConfig
```

Comment:
Can we make a `VariantName` type to make the `map[string]` more obvious with `map[VariantName]`?